src/DatabaseLibrary/connection_manager.py

```diff
@@ -50,7 +50,7 @@ def connect_to_database(self, dbapiModuleName=None, dbName=None, dbUsername=None
         your database credentials.
 
         Example db.cfg file
-        | [default]
+        | [alias]
```

Comment:
Nice - thanks for that, we are keeping previous file configuration while allowing to use it for multiple aliased connections.